package.json

```diff
@@ -1,76 +1,82 @@
+    "@types/node": "12.6.8",
+    "@types/react": "16.8.23",
+    "@types/react-dom": "16.8.4",
     "classnames": "^2.2.6",
-    "eslint": "^4.19.1",
-    "eslint-config-prettier": "^2.9.0",
-    "eslint-config-react-app": "^2.1.0",
-    "eslint-plugin-flowtype": "^2.34.1",
-    "eslint-plugin-import": "^2.6.0",
-    "eslint-plugin-jest": "^21.18.0",
-    "eslint-plugin-jsx-a11y": "^5.1.1",
-    "eslint-plugin-prettier": "^2.6.1",
-    "eslint-plugin-react": "^7.1.0",
-    "husky": "^0.14.3",
-    "jest-junit": "^5.1.0",
-    "lint-staged": "^7.2.0",
-    "node-sass-chokidar": "^1.3.3",
+    "enzyme": "^3.10.0",
+    "enzyme-adapter-react-16": "^1.14.0",
+    "eslint-config-prettier": "^6.0.0",
+    "eslint-plugin-jest": "^22.6.4",
+    "eslint-plugin-prettier": "^3.1.0",
+    "husky": "^2.4.0",
+    "jest-enzyme": "^7.0.2",
+    "jest-junit": "^6.4.0",
+    "lint-staged": "^8.2.0",
+    "node-sass-chokidar": "^1.3.5",
     "npm-run-all": "^4.1.3",
-    "prettier": "^1.13.7",
-    "react": "^16.4.1",
-    "react-dom": "^16.4.1",
-    "react-scripts-ts": "^2.17.0",
-    "serve": "^9.4.0",
-    "stylelint": "^9.4.0",
-    "stylelint-config-recommended-scss": "^3.2.0",
-    "stylelint-config-standard": "^18.2.0",
+    "prettier": "1.18.2",
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6",
+    "react-scripts": "3.0.1",
+    "serve": "^11.0.1",
+    "shelljs": "^0.8.2",
+    "stylelint": "^10.0.1",
+    "stylelint-config-recommended-scss": "^3.3.0",
+    "stylelint-config-standard": "^18.3.0",
     "stylelint-declaration-use-variable": "^1.6.1",
-    "stylelint-scss": "^3.2.0",
-    "tslint": "^5.11.0",
-    "tslint-config-prettier": "^1.15.0",
-    "tslint-react": "^3.6.0",
-    "typescript": "^3.0.3"
+    "stylelint-scss": "^3.8.0",
+    "typescript": "^3.5.3"
   },
   "scripts": {
-    "start": "npm-run-all -p start:css start:ts",
-    "start:css": "npm-run-all build:css build:css:watch",
-    "start:ts": "react-scripts-ts start",
-    "build": "npm-run-all build:css build:ts",
-    "build:css": "node-sass-chokidar src/ -o src/",
-    "build:css:watch": "node-sass-chokidar src/ -o src/ --watch --recursive",
-    "build:ts": "react-scripts-ts build",
+    "prestart": "node bin/pre-start",
+    "start": "react-scripts start",
+    "build": "react-scripts build",
     "lint": "npm-run-all -c lint:*",
-    "lint:eslint": "eslint 'src/**/*.{js,jsx}'",
-    "lint:prettier": "prettier --write **/*.{js,jsx,ts,tsx,css,scss,json,md}",
+    "lint:eslint": "eslint 'src/**/*.{ts,tsx}'",
     "lint:stylelint": "stylelint src/**/*.scss",
-    "lint:tslint": "tslint 'src/**/*.{ts,tsx}'",
-    "pretest": "npm-run-all build:css",
-    "test": "react-scripts-ts test --env=jsdom",
-    "pretest:coverage": "npm-run-all build:css",
-    "test:coverage": "react-scripts-ts test --env=jsdom --coverage",
-    "eject": "react-scripts-ts eject",
-    "precommit": "lint-staged",
+    "lint:prettier": "prettier --write **/*.{ts,tsx,json,scss,md}",
```

Comment:
When I run `yarn lint` it ends up writing modifications to `.github/PULL_REQUEST_TEMPLATE.md`. Should we commit those changes as part of this PR?

```diff
diff --git a/.github/PULL_REQUEST_TEMPLATE.md b/.github/PULL_REQUEST_TEMPLATE.md
index f485971..20c4e24 100644
--- a/.github/PULL_REQUEST_TEMPLATE.md
+++ b/.github/PULL_REQUEST_TEMPLATE.md
@@ -1,29 +1,28 @@
-Problem
-=======
+# Problem
+
 problem statement, including
 [link to Pivotal Tracker #12345678](https://www.pivotaltracker.com/story/show/12345678)
 
-Solution
-========
+# Solution
+
 What I/we did to solve this problem
 
 with @pairperson1
 
-Change summary:
----------------
-* Tidy, well formulated commit message
-* Another great commit message
-* Something else I/we did
+## Change summary:
+
+- Tidy, well formulated commit message
+- Another great commit message
+- Something else I/we did
+
+## Steps to Verify:
 
-Steps to Verify:
-----------------
 1. A setup step / beginning state
 1. What to do next
 1. Any other instructions
 1. Expected behavior
 1. Suggestions for testing
 
-Screenshots (optional):
------------------------
-Show-n-tell images/animations here
+## Screenshots (optional):
 
+Show-n-tell images/animations here
```